conf/evolutions/default/1.sql

```diff
@@ -0,0 +1,16 @@
+-- Products schema
+
+-- !Ups
+
+CREATE TABLE IF NOT EXISTS Product (
+    id          UUID          NOT NULL PRIMARY KEY,
+    name        VARCHAR(100)  NOT NULL,
+    quantity    INTEGER,
+    price       DECIMAL,
+    created_at  TIMESTAMP,
+    updated_at  TIMESTAMP
```

Comment:
timestamp should never be null (add constraint). Additionally set the default value to be the current timestamp

Reply:
Added `NOT NULL` constraint to the timestamps and appended `DEFAULT CURRENT_TIMESTAMP` so the default is 'now'.